Mage.Sets/src/mage/cards/f/FranticInventory.java

```diff
@@ -13,6 +15,7 @@
 
 /**
  * @author TheElk801
+ * modified tiera3 - added cardHint
```

Comment:
(fyi no need to mention this unless major reimplementation - git tracks changes line by line so it's kinda useless info anyway)